appveyor.yml

```diff
@@ -0,0 +1,43 @@
+version: 1.1.x.{build}
+skip_commits:
+  message: /(^\(?doc\)?.*|.*[A|a]cceptance [T|t]est.*)/
+clone_depth: 10
+init:
+- SET
+- 'mkdir C:\ProgramData\PuppetLabs\code && exit 0'
+- 'mkdir C:\ProgramData\PuppetLabs\facter && exit 0'
+- 'mkdir C:\ProgramData\PuppetLabs\hiera && exit 0'
+- 'mkdir C:\ProgramData\PuppetLabs\puppet\var && exit 0'
+environment:
+  matrix:
+  - PUPPET_GEM_VERSION: 3.7.5
+    RUBY_VER: 193
+  - PUPPET_GEM_VERSION: 3.7.5
+    RUBY_VER: 200-x64
+  - PUPPET_GEM_VERSION: 3.8.1
+    RUBY_VER: 193
+  - PUPPET_GEM_VERSION: 3.8.1
+    RUBY_VER: 200-x64
+  - PUPPET_GEM_VERSION: 4.0.0
+    RUBY_VER: 21
+  - PUPPET_GEM_VERSION: 4.0.0
+    RUBY_VER: 21-x64
+matrix:
+  fast_finish: true
+install:
+- SET PATH=C:\Ruby%RUBY_VER%\bin;%PATH%
+- gem install bundler --quiet --no-ri --no-rdoc
+- bundle install --jobs 4 --retry 2 --without system_tests
+- type Gemfile.lock
+build: off
+test_script:
+- bundle exec puppet -V
+- ruby -v
+- bundle exec rake spec SPEC_OPTS='--format documentation'
```

Comment:
No it shouldn't there is a particular rake task in rspec-puppet called spec if on the other hand it was bundle exec rspec spec then yes it would be calling beaker-rspec.